egs/tedlium/s5_r2/local/run_learn_lex.sh

```diff
@@ -0,0 +1,79 @@
+#! /bin/bash
+#
+# This script demonstrates a lexicon learning recipe, which aims to imrove
+# the pronounciation of abbreviated words in the TED-LIUM lexicon. It assumes
+# the model exp/tri3 already exists. Please see steps/dict/learn_lexicon.sh
+# for explanation of the options. 
+#
+# Copyright 2016  Xiaohui Zhang
+# Apache 2.0
+
+. cmd.sh
+. path.sh
+
+oov_symbol="<unk>"
+prior_counts="2.5-1-1"
+min_prob=0.3
+var_mass=0.7
```

Comment:
need comments on all of these variables.  var_mass might not be good name.. self-explanatory names are better.
As separators for strings, commas are better. minus signs could easily be confused with minus signs in numbers e.g. 1.0e-04.